projects/packages/search/src/dashboard/components/pages/upsell-page.scss

```diff
@@ -28,5 +28,5 @@
 }
 
 .price-tip-icon {
-	fill: $studio-gray-10;
+	fill: $studio-gray-10 !important;
```

Comment:
@dognose24: Let's avoid introducing `!important` to our styles when increasing selector specificity will do.

In this case, replacing `.price-tip-icon` with `.price-tip .price-tip-icon` will do.